pkg/relayer/cmd/cmd.go

```diff
@@ -84,7 +84,8 @@ for such operations.`,
 	cmd.Flags().Bool(cosmosflags.FlagGRPCInsecure, true, "Used to initialize the Cosmos query context with grpc security options. It can be used to override the `QueryNodeGRPCInsecure` field in the config file if specified.")
 	cmd.Flags().String(cosmosflags.FlagChainID, "pocket", "The network chain ID")
 	cmd.Flags().StringVar(&flagLogLevel, cosmosflags.FlagLogLevel, "debug", "The logging level (debug|info|warn|error)")
-	cmd.Flags().Float64(cosmosflags.FlagGasAdjustment, 1.5, "The adjustment factor to be multiplied by the gas estimate returned by the tx simulation")
+	// Transactions submitted by the RelayMiner are important (i.e. responsible for claims and proofs) that we are willing to overpay on gas fees
```

Comment:
Is it worth mentioning that typically we would use 1.5, so that the contrast is clear?